metricflow-semantics/metricflow_semantics/test_helpers/snapshot_helpers.py

```diff
@@ -165,3 +175,167 @@ def add_overwrite_snapshots_cli_flag(parser: _pytest.config.argparsing.Parser) -
         action="store_true",
         help="Overwrites existing snapshots by ones generated during this testing session",
     )
+
+
+# In plan outputs, replace strings that vary from run to run with this so that comparisons can be made
+# consistently.
+PLACEHOLDER_CHAR_FOR_INCOMPARABLE_STRINGS = "*"
+
+
+def make_schema_replacement_function(system_schema: str, source_schema: str) -> Callable[[str], str]:
```

Comment:
I assume this is all just a straight copy/paste with maybe some autoformatting applied, but nothing else is changed. If that isn't correct I should probably read these files.

Reply:
Yep, that's correct.